buildtest/settings/settings.schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://raw.githubusercontent.com/HPC-buildtest/buildtest-framework/devel/buildtest/default.schema.json",
+  "$id": "https://raw.githubusercontent.com/HPC-buildtest/buildtest-framework/devel/buildtest/settings.schema.json",
```

Comment:
Was this changed in the schemas repo?

Reply:
right now this schema is in buildtest-framework as posed to schema repo. And yes this file has been updated, you can see in my branch the file is called ``settings.schema.json`` once its merged it will be in the devel location. 
https://github.com/shahzebsiddiqui/buildtest-framework/blob/regtest_fixups/buildtest/settings/settings.schema.json

Reply:
Can we update the schema repository before committing changes here?

Reply:
Recall the ``default.schema.json`` was originally part of buildtest-framework, all i did was just rename the file to comply with the correct term. Question is does it make sense to put this in schema given that only buildtest-framework will use it to validate configuration. If we put in schema which i don't really have any strong opinion where would we place it.

Reply:
Also recall this schema won't be using any versions like ``script`` and other schema types. This just for validating the buildtest setting that each site will need to do to get buildtest configured.

Reply:
Ah okay, then we don't need to do that.